canonical-web-and-design/ubuntu.com
# Rename telecommunications to telco

## Done
Rename telecommunications to telco

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Check the page renders
- Check /telcos and /telecommunications redirect to /telco

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3167
